### ISSUE TITLE
Unsupported on Python 3.10

### ISSUE THREAD

**Issue by aidenmitchell** [lobe/lobe-python#32]
I have a working Lobe install on Python 3.9.5. Attempting `pip install lobe[tf]` on Python 3.10.2 fails with the following error:
```
ERROR: Cannot install lobe[tf]==0.2.0, lobe[tf]==0.2.1, lobe[tf]==0.3.0, lobe[tf]==0.4.0, lobe[tf]==0.5.0, lobe[tf]==0.6.0 and lobe[tf]==0.6.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    lobe[tf] 0.6.1 depends on tensorflow~=2.5.0; platform_machine != "armv7l" and extra == "tf"
    lobe[tf] 0.6.0 depends on tensorflow~=2.5.0; platform_machine != "armv7l" and extra == "tf"
    lobe[tf] 0.5.0 depends on tensorflow~=2.5.0; platform_machine != "armv7l" and extra == "tf"
    lobe[tf] 0.4.0 depends on onnxruntime~=1.7.0; platform_machine != "armv7l"
    lobe[tf] 0.3.0 depends on tensorflow==2.4; platform_machine != "armv7l"
    lobe[tf] 0.2.1 depends on tensorflow<2 and >=1.15.2; platform_machine != "armv7l"
    lobe[tf] 0.2.0 depends on tensorflow<2 and >=1.15.2; platform_machine != "armv7l"

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

It seems that `tensorflow==2.5.0` fails as well:
```
ERROR: Could not find a version that satisfies the requirement tensorflow==2.5.0 (from versions: 2.8.0rc1, 2.8.0)
ERROR: No matching distribution found for tensorflow==2.5.0
```

Any thoughts? It's possible I've misconfigured something here.

**Comment by mbeissinger** [comment]
Looks like tensorflow would be the base required here -- I'll try bumping to tf 2.8 which is supposed to support python 3.10

**Comment by mbeissinger** [comment]
Should be fixed by #33
